TrackerConditions/src/AlignedTrackerMaker.cc

```diff
@@ -113,7 +113,7 @@ namespace mu2e {
 
   // now set the aligned Panel midpoint and direction
   panel._straw0Direction = panel.getStraw(0).getDirection();
-  panel._straw0MidPoint = aligned_straw0MidPoint;
+  panel._straw0MidPoint = aligned_straw0MidPoint / StrawId::_nlayers;
```

Comment:
Should this data member be renamed panelMidPoint?  Does this division bring the midpoint into the plane between the straw layers?  I'm not sure how that works.

Reply:
Oh goodness - hold on a minute. I had another look at fillPointers. 
I misinterpreted what straw0MidPoint was. I should be doing exactly this: https://github.com/Mu2e/Offline/blob/5bd6522f42531a7fb91f99b484aca4e471b62625/TrackerGeom/src/Panel.cc#L46-L52

I'll fix this now

Reply:
OK, fixed this